common/app/views/support/Commercial.scala

```diff
@@ -224,8 +224,11 @@ object Commercial {
       ) mkString " | "
     }
 
-    def mkInteractionTrackingCode(containerIndex: Int, cardIndex: Int, card: ContentCard): String = Seq(
+    def mkInteractionTrackingCode(containerIndex: Int, cardIndex: Int, card: ContentCard, containerDisplayName: Option[String])(implicit request: RequestHeader): String = Seq(
+      containerDisplayName.getOrElse("unknown container"),
+      Edition(request).id,
```

Comment:
No you're right - just looked at it again and I confused the ordering - will change.